.pre-commit-config.yaml

```diff
@@ -0,0 +1,42 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
no need of this hooks from pre-commit, this will be handled by black

Reply:
there are some python specific, let it be they don't consume extra resouces